doc/pages/administrator/066-migrate_to_astartedefaultingress.md

```diff
@@ -0,0 +1,130 @@
+# Migrating to the Astarte Default Ingress
+
+If your Astarte deployment is still being served by the deprecated AstarteVoyagerIngress, it is
+highly advised to move to the new managed ingress, namely, the AstarteDefaultIngress.
+
+The current page focuses on describing the procedure for migrating your ingress with ease, simply
+employing `astartectl`. 
+
+Please, ensure to read the entirety of this page and to understand all the concepts and the
+implications before performing the actual migration.
+
+## Prerequisites and preliminary checks
+
+Before starting with the actual migration procedure, some preliminary activities are required:
+
+- ensure that the version of the Astarte operator in your cluster is at least `>= v1.0.1` and
+  stable. If this requirement is not fulfilled, please refer to the [Upgrade
+  Procedures](000-upgrade_index.html) section;
+
+- the [`ingress-nginx`](https://kubernetes.github.io/ingress-nginx/) ingress controller must be
+  deployed in your cluster (see [this section](020-prerequisites.html#nginx) for the details).
+  Ensure to take note of the `ingress-class` of the controller which is meant to handle your Astarte
+  ingress. This information will come handy during the migration itself;
+
+- ensure to deploy the TLS secrets used to secure the communications to and from Astarte. For the
+  details, please refer to the [Handling Astarte certificates](050-handling_certificates.html) page;
+
+- ensure that [`astartectl`](https://github.com/astarte-platform/astartectl) is installed on your
+  machine and its version is at least `>= v1.0.0`.
+
+## Performing the migration
+
+Performing the actual ingress migration is as simple as executing an `astartectl` command:
+
+```bash
+$ astartectl cluster instances migrate-ingress \
+  --namespace <astarte-namespace> \
+  --operator-name <astarte-operator-name> \
+  --operator-namespace <astarte-operator-namespace>
+```
+
+To successfully complete the procedure, you will be prompted to insert information such as:
+- the name of the Astarte TLS secrets, 
+- the name of the to-be-installed AstarteDefaultIngress resource,
+- the ingress class of the NGINX ingress controller.
+
+Before starting the migration routine, you will be asked to review the generated
+AstarteDefaultIngress custom resource. The migration will start only upon confirmation.
```

Comment:
What if the user wants to add an annotation to the generated ADI CR before confirming it?
For example by migrating manually an annotation that was present in the AVI CR?